File: Lab2/javaServer/src/main/java/com/JabaJabila/javaServer/models/OwnerCreationInfo.java
package com.JabaJabila.javaServer.models;

import java.util.Date;

public class OwnerCreationInfo {
    private String name;
    private Date birthdate;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Date getBirthdate() {
        return birthdate;
    }

    public void setBirthdate(Date birthdate) {
        this.birthdate = birthdate;
    }
}

File: Lab2/Codegen/MyHttpClient/GeneratedModels/OwnerCreationInfo.cs
namespace MyHttpClient.GeneratedModels;
public class OwnerCreationInfo
{
    public OwnerCreationInfo(string name, string birthdate)
    {
        Name = name;
        Birthdate = birthdate;
    }

    public string Name { get; set; }

    public string Birthdate { get; set; }
}